includes/class-wc-stripe-feature-flags.php

```
@@ -98,34 +98,13 @@ public static function is_amazon_pay_available() {
-
-		/**
-		 * Filter to control the availability of the Stripe Checkout Sessions feature.
-		 *
-		 * @since 10.4.0
-		 * @deprecated This filter will be removed in version 10.8.0. Stripe Checkout Sessions is permanently available as of version 10.6.0.
-		 * @param bool $is_checkout_sessions_available Whether Stripe Checkout Sessions should be available.
-		 */
-		return (bool) apply_filters( 'wc_stripe_is_checkout_sessions_available', true );
+		wc_deprecated_function( __METHOD__, '10.9.0', 'WC_Stripe_Helper::is_checkout_sessions_available()' );
+		return WC_Stripe_Helper::is_checkout_sessions_available();
 	}
```

PHPMD (2.15.0)

[error] The method is_checkout_sessions_available is not named in camelCase. (undefined)

(CamelCaseMethodName)